verification/src/http_server/handlers/verification/solidity/version_list.rs

```diff
@@ -0,0 +1,21 @@
+use super::types::VersionsResponse;
+use crate::{
+    compiler::{download_cache::DownloadCache, fetcher::Fetcher},
+    solidity::github_fetcher::GithubFetcher,
+};
+
+use actix_web::{
+    web::{self, Json},
+    Error,
+};
+
+pub async fn get_version_list(
+    cache: web::Data<DownloadCache<GithubFetcher>>,
+) -> Result<Json<VersionsResponse>, Error> {
+    let mut versions = cache.fetcher().all_versions();
+    // sort in descending order
+    versions.sort_by(|x, y| y.cmp(x));
```

Comment:
`versions.sort()` ?

Reply:
by defalut it will sort in ascending order

Reply:
oh, I missed that. may be it would be more readable to do `x.cmp(y).reverse()`, because it is very easy to miss, that it's `y.cmp(x)`, and not `x.cmp(y)` https://doc.rust-lang.org/std/cmp/enum.Ordering.html#method.reverse